decidim-comments/app/frontend/comments/add_comment_form.component.jsx

```diff
@@ -137,7 +139,8 @@ const AddCommentFormWithMutation = graphql(gql`
           body,
           author: {
             __typename: 'Author',
-            name: ownProps.currentUser.name
+            name: ownProps.currentUser.name,
+            avatarUrl: defaultAvatarUrl
```

Comment:
I'd add a avatarUrl to currentUser at the api and use it here instead of having a defaultAvatarUrl